.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -4,7 +4,7 @@ labels: "bug"
 body:
 - type: markdown
   attributes:
-    value: Before raising an issue, please search for existing issues to avoid creating duplicates. For questions and support please use the [community forum](https://community.gitpod.io/).
+    value: Before raising an issue, please search for existing issues to avoid creating duplicates. For questions and support please use the [community discord server](https://www.gitpod.io/chat).
```

Comment:
Delete this entire file so that the files within gitpod-io/.github becomes the single source of truth? Adjust gitpod-io/.github if required.

Reply:
There are a couple things in this `bug_report.yml` which are not in the org level one.



```yaml
- type: input
  id: workspace
  attributes:
    label: Workspace affected
    description: Does this issue affect a specific workspace?
    placeholder: e.g. kumquat-goose-831fpnxf
```

Reply:
FYI, `gitpod-io/gitpod` has also enabled GitHub Issues[[1](https://gh-community.github.io/issue-template-feedback/structured/)] which is not enabled for other repositories. I'd vote for keeping the bug report issue template changes in this repository if needed to keep issue forms enabled. 💡

Reply:
Thank you, @gtsiolis ! Appreciate the heads up. :smiley: 

@ghuntley let's plan on continuing to use these repo level template files for the short term (that way we can get folks to Discord). Long term, I'll create a new issue - I'm not sure how best to use Github org level templates (to both consolidate templates, but also override the org level ones, when needed).

Reply:
Opened #7415